types/react-loadable/index.d.ts

```diff
@@ -24,64 +25,76 @@ export interface CommonOptions {
      * (this is considered a valid React component).
      */
     // NOTE: () => null is only needed until React.SFC supports components returning null
-    LoadingComponent: React.ComponentClass<LoadingComponentProps> | React.SFC<LoadingComponentProps> | (() => null);
+    loading: React.ComponentClass<LoadingComponentProps> | React.SFC<LoadingComponentProps> | (() => null);
```

Comment:
Thanks @Kovensky - I wasn't aware of `React.ComponentType`. I made that change (and a couple other minor ones), would you mind taking another look when you have the chance?